docs/authentication/social-connections/xero.mdx

```diff
@@ -27,10 +27,10 @@ First, you need to create a new OAuth2 Xero app. Click New App and fill all the
 
 ![Creating a new application](/docs/images/authentication-providers/xero/creating-new-application.jpg)
 
-In the Clerk Dashboard, go to **User & Authentication > [Social Connections](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections)** and enable Xero. In the modal that opened, ensure **Use custom credentials** is enabled and copy **Authorized redirect URI**. Make sure the value matches the **Redirect URIs**, as shown below.
+Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **Xero**. Toggle on **Use custom credentials** and copy **Authorized redirect URI**. Make sure the value matches the **Redirect URIs**, as shown below.
 
 ![Copying values from the Xero dashboard](/docs/images/authentication-providers/xero/copying-values-from-xero-dashboard.jpg)
 
 Copy the **Client ID** and **Client secret** as shown in the above image. Go back to the Clerk Dashboard and paste them into the respective fields.
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with Xero is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with Xero is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with Xero is now configured for your instance.
```